### ISSUE TITLE
Homebrew version not compatible with pandoc 2.13

### ISSUE THREAD

**Issue by danrot** [lierdakil/pandoc-crossref#301]
I am having some issues when using the version of pandoc-crossref from homebrew. The latest version 0.3.10.0a (that's where pandoc 2.13 support has landed, right?) seems not to be available on [homebrew](https://formulae.brew.sh/formula/pandoc-crossref#default), which only shows 0.3.10.0, and now I am having some issues with the bibliography... Has the update of the homebrew package been forgotten?

**Comment by lierdakil** [comment]
Hi. I'm not managing homebrew releases. Or indeed any releases besides Hackage and this repo.

0.3.10.0a is basically the same as 0.3.10.0, i.e. 0.3.10.0a is just a git tag with an updated CI config (actually, there were some internal changes between these particular releases, but none of those should affect the observable behaviour).

If homebrew release complains about pandoc versions, it just needs to be rebuilt against newer pandoc, which I think homebrew manages somehow, but I don't know how it works.

Honestly, not a lot has changed between pandoc 2.12 and 2.13, so I'm doubtful your bibliography issues are due to the version mismatch. Can you describe the issue in more detail?

**Comment by danrot** [comment]
Sorry, didn't know that, and thought this would be the right place to ask... But what you are saying is interesting, going to explain the problem in more detail, maybe the error message I see was a red herring.

I am building my document using the following command:

```bash
	pandoc\
	    --standalone\
	    --toc\
	    --number-sections\
	    --citeproc\
	    --csl https://raw.githubusercontent.com/citation-style-language/styles/master/ieee.csl\
	    --bibliography sources.bib\
	    -V geometry:margin=2cm\
	    -V lof\
	    --filter pandoc-crossref\
	    -o script.pdf\
	    script.md
```

What made me skeptical at some point was the following that is shown during the processign:

```
WARNING: pandoc-crossref was compiled with pandoc 2.12 but is being run through 2.13. This is not supported. Strange things may (and likely will) happen silently.
```

Especially the last sentence got my attention. I've tried multiple things, and either the citings (e.g. `[@pandoc2021]`) were not replaced according to my citation style, or the bibliography at the end was not generated.

But after trying a bit further after you've said that this is likely not the issue, I digged a bit further, and realized that you've described [in your documentation regarding using pandoc-crossref with pandoc-citeproc](https://lierdakil.github.io/pandoc-crossref/#pandoc-citeproc-and-pandoc-crossref) the order of the execution matters. I didn't read that very carefully, because I didn't use pandoc-citeproc, since it is deprecated and homebrew will not even let me install it.

However, I still tried to change the order of the arguments, and now it is working. Seems like the `--filter pandoc-crossref` parameter still has to be before `--citeproc`. I imagined that only the filters themeselves respect the order... Should that part of the documentation maybe be updated?

Anyway, thanks for your help, it seems to be working now 🙂 

```bash
	pandoc\
	    --standalone\
	    --toc\
	    --number-sections\
	    --filter pandoc-crossref\
	    --citeproc\
	    --csl https://raw.githubusercontent.com/citation-style-language/styles/master/ieee.csl\
	    --bibliography sources.bib\
	    -V geometry:margin=2cm\
	    -V lof\
	    -o script.pdf\
	    script.md
```
